types/redux-actions/index.d.ts

```diff
@@ -85,7 +89,7 @@ export function createAction<Payload, Arg1, Arg2, Arg3, Arg4>(
 
 export function createAction<Payload>(
     actionType: string
-): ActionFunctionAny<Action<Payload>>;
+): ActionFunction1<Payload, Action<Payload>>;
```

Comment:
This can't be right -- now there are 2 single-argument and they return different results.